autoload/ctrlp.vim

```diff
@@ -791,7 +791,7 @@ fu! s:BuildPrompt(upd)
 		exe 'echoh' hibase '| echon "_" | echoh None'
 	en
 	if a:upd
-		cal s:notifyChange()
+		cal s:notifySearch()
```

Comment:
Probably `s:NotifySearch()` ? (as far as I can see the kien's code style)

Reply:
done